### DISCUSSION TITLE
Border child open by default?

### DISCUSSION THREAD

**Discussion by M-Willett** [caplin/FlexLayout#321] [Q&A] [answered]
Is it possible to have a border pane open by default? 
Desired behaviour is to create a navigation component in an IDE much like the left sidebar in VS Code (for example: File explorer, Search, Source Control, etc).

To better illustrate, having the highlighted area in the screenshot visible on page load, by default.
![image](https://user-images.githubusercontent.com/2579403/175507838-5c36bbc4-861b-4577-b09b-a5af7034a3f8.png)

**Comment by nealus** [comment, marked as answer]
just set the selected property in the border to 0:

`		{
			"type": "border",
			"selected": 0,
			"location": "left",
			"children": [
				{
					"type": "tab",
					"id": "#30b43eab-2da6-4f10-b434-3b0fa3d847c8",
					"name": "Navigation",
					"altName": "The Navigation Tab",
					...`

**Reply by M-Willett** [reply]
Works! Thank you.
